### PR TITLE
nettest/netlog: modify netlog TC

### PR DESCRIPTION
**Pull request by pillip8282**
TC pass was checked by returning value of NET_LOG API. However
previous API doesn't return correct value so TC couldn't check
validation of API. After modify APIs then change expected value of TC.

### CODE REVIEW

File: external/netutils/netlib_netlog.c [hunk 1]
```diff
@@ -84,24 +84,25 @@ int netlogger_print(netlog_module_e mod,
 		return 0;
 	}
 
+  int len = 0;
```
Comment (jeongchanKim) [inline external/netutils/netlib_netlog.c:87]: Please check the indentation:)

File: apps/examples/nettest/netlog_test.c [hunk 2]
```diff
@@ -103,20 +87,13 @@ END_TEST_F
...
-  int len = NET_LOGV(NL_MOD_WIFI_MANAGER, wifimanager_verb_log);
-  ST_EXPECT_EQ(19, len);
-
-  res = netlog_set_color(NL_MOD_NETLIB, NL_COLOR_YELLOW);
-  ST_EXPECT_EQ(0, res);
-  res = netlog_set_level(NL_MOD_NETLIB, NL_LEVEL_VERB);
-  ST_EXPECT_EQ(0, res);
-  len = NET_LOGV(NL_MOD_NETLIB, netlib_verb_log);
-  ST_EXPECT_EQ(13, len);
+  ST_EXPECT_EQ(0, netlog_reset());
```
Comment (dineshdas-samsung) [inline apps/examples/nettest/netlog_test.c:90]: Can we align this please?

File: external/netutils/netlib_netlog.c [hunk 3]
```diff
@@ -66,7 +66,7 @@ static inline void _print_time(void)
 	clock_gettime(CLOCK_REALTIME, &current_time);
 
 	unsigned int micro_sec = current_time.tv_nsec / 10000000;
-	printf(" [%02d:%02d:%02d.%02u]",
+	return printf(" [%02d:%02d:%02d.%02u]",
```
[the remark this answers is not in the document]
Comment (pillip8282) [reply external/netutils/netlib_netlog.c:69]: Yes right.

File: external/netutils/netlib_netlog.c (hunk 1 above)
Comment (pillip8282) [reply external/netutils/netlib_netlog.c]: Thanks I'll modify

**Review by an4967 — APPROVED** [review]
LGTM

**Pull request merged**